README.md

```diff
@@ -247,14 +247,23 @@ This assumes that the `etc` directory contains the certificate, truststore, and
 
 [Bug Tracker](https://github.com/symphonyoss/bot-unfurl/issues)
 
-## License
+## Contributing
+
+1. Fork it (<https://github.com/symphonyoss/bot-unfurl/fork>)
+2. Create your feature branch (`git checkout -b feature/fooBar`)
+3. Read our [contribution guidelines](.github/CONTRIBUTING.md) and [Community Code of Conduct](https://www.finos.org/code-of-conduct)
+4. Commit your changes (`git commit -am 'Add some fooBar'`)
+5. Push to the branch (`git push origin feature/fooBar`)
+6. Create a new Pull Request
 
-Copyright 2016 Fintech Open Source Foundation
+## License
 
-Distributed under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0).
+The code in this repository is distributed under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0).
 
 SPDX-License-Identifier: [Apache-2.0](https://spdx.org/licenses/Apache-2.0)
 
+Copyright 2016-2019 Symphony LLC
```

Comment:
This copyright notice is incorrect in (at least) two ways:

1. It only requires a year of first publication - [copyrights are inherently open ended into the future](https://en.wikipedia.org/wiki/Copyright_notice#Form_of_notice_for_visually_perceptible_copies)
2. The copyright holder of this code is FINOS, _not_ Symphony LLC, and as the original author I am opposed to any reassignment of copyright away from FINOS (if that's what's being proposed)